roles/zuul/files/jobs/dib.yaml

```diff
@@ -0,0 +1,38 @@
+- publisher:
```

Comment:
This publisher is already defined in bonnyci.yaml, I don't think we need to redefine it here.